src/HotChocolate/Core/test/Validation.Tests/FragmentSpreadIsPossibleRuleTests.cs

```diff
@@ -58,21 +48,13 @@ public void InterfaceTypeDoesMatch()
                     name
                 }
             ");
-            context.Prepare(query);
-
-            // act
-            Rule.Validate(context, query);
-
-            // assert
-            Assert.Empty(context.Errors);
         }
 
         [Fact]
         public void UnionTypeDoesMatch()
```

Comment:
@michaelstaib  Can this really work?

Reply:
This one ... yes since pet implements dog.

Reply:
I mean dog implements pet :)